src/bundler.ts

```diff
@@ -370,10 +370,12 @@ export class Bundler {
     let template = dom5.query(domModule, matchers.template);
     if (!template) {
       template = dom5.constructors.element('template')!;
-      dom5.append(domModule, template);
+      treeAdapters.default.setTemplateContent(
+          template, dom5.constructors.fragment());
+      astUtils.prepend(domModule, template);
```

Comment:
You can see the effects in one build condition here:
https://github.com/Polymer/polymer-cli/issues/614